Makefile

```diff
@@ -43,34 +43,34 @@ clean: ## Cleans the underlying build.
 runconformance: generate $(CONNECT_CONFORMANCE) ## Run the new conformance test suite.
 	./gradlew $(GRADLE_ARGS) conformance:client:google-java:installDist conformance:client:google-javalite:installDist
 	$(CONNECT_CONFORMANCE) -v --mode client --conf conformance/client/lite-unary-config.yaml \
-		--known-failing @conformance/client/known-failing-unary-cases.txt -- \
+		--known-failing @conformance/client/known-failing-unary-cases.txt --trace -- \
```

Comment:
Perhaps we could set `CONNECT_CONFORMANCE_ARGS ?= -v --mode client --trace` which would allow easily running with additional options if needed.

Reply:
Ah, nice. I like it.

Done.